CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.4.0 (2018-11-06)
+--------------------------
+
+- Changes license to MIT.
```

Comment:
You can also add 

```
- Uses standardised REANA component documentation structure.
```

as for the REANA-Message-Broker component.

Reply:
Added